runner_manager/backend/aws.py

```diff
@@ -0,0 +1,81 @@
+from typing import Dict, List, Literal
+
+from boto3 import client
+from botocore.exceptions import ClientError
+from pydantic import Field
+
+from runner_manager.backend.base import BaseBackend
+from runner_manager.logging import log
+from runner_manager.models.backend import AWSConfig, AWSInstanceConfig, Backends
+from runner_manager.models.runner import Runner
+
+
+class AWSBackend(BaseBackend):
+    name: Literal[Backends.aws] = Field(default=Backends.aws)
+    config: AWSConfig
+    instance_config: AWSInstanceConfig
+
+    @property
+    def client(self) -> client:
+        """Returns a AWS Compute Engine client."""
```

Comment:
```suggestion
        """Return a AWS Compute Engine client."""
```
Make sure it's consistent elsewhere (either put 's' or don't)